### PR TITLE
Let's get this (shopping) party started

### LINKED ISSUE #63: Revenue information on Gatsby store product page
@fk @jlengstorf 

From our meeting this morning Flo mentioned that it may be a good idea to include a blurb on the store that any revenue we get from the swag store is going to be put back in the open source community.

My suggested wording is here:

> We're happy to have you here at Gatsby, and thanks so much for helping us out. Any money made from your purchase at our swag store goes right back in to the open source community to help with Gatsby events and workshops.

### LINKED ISSUE #65: Add favicons to the store
Let's use <https://realfavicongenerator.net> for this. Implementation [on marisamorby.com](https://github.com/jlengstorf/marisamorby.com/blob/master/src/components/shared/Layout.js#L40-L47) can serve as an example here.

### LINKED ISSUE #70: Design system: Update/expand color palette
Designs/mocks use the colors from gatsbyjs.org, current palette is a little off.

### LINKED ISSUE #73: Update cart component
* [x] adjust design
* [x] add "empty state" (sad cart illustration + "Your cart is sad" copy)
* [x] add "added to cart" component
* [x] add "continue shopping" link below "Check Out" button
* [x] add close button next to "Your Cart" headline
* [x] update existing components to fit design

image

image

### LINKED ISSUE #74: Add "About the Gatsby Store" section
image

### LINKED ISSUE #77: Update product component
* [x] thumbnail button style
* [x] `<select>` styles
* [x] headline and price styles

image

### LINKED ISSUE #84: Add "Futura PT" webfont
Currently Futura PT as referenced in https://github.com/gatsbyjs/store.gatsbyjs.org/blob/7708ffdfa7a1b805aa5cf7d2257067a5b2c22977/src/utils/styles.js#L65 is only seen by people who have the font installed locally.

We got a license for gatsbyjs.org, and we're using it for next.gatsbyjs.org, so I assume using it on another subdomain should be fine.

### PR DESCRIPTION
**Pull request by fk**
* [x] add favicons to the store, fixes #65 (https://realfavicongenerator.net/favicon_checker?protocol=https&site=deploy-preview-86--gatsby-store.netlify.com%2F#.W0i8ydgzbXE)
* [x] update cart design; fixes #73
* [x] add "About the Gatsby Store" section, fixes #74, fixes #63
* [x] update product card design; fixes #77
* [x] add the "Futura PT" webfont; fixes #84 
* [x] basic updates to the discount code display component, ref. #85

Also fixes #70.

### CODE REVIEW

File: src/components/ProductPreview/ProductPreview.js [hunk 1]
```diff
@@ -67,8 +75,8 @@ export default ({ product }) => (
   <Preview>
     <ProductImages images={product.images} alt={product.title} />
     <Name>{product.title}</Name>
+    <Price>${product.variants[0].price}</Price>
```
Comment (jlengstorf) [inline src/components/ProductPreview/ProductPreview.js]: Should we include "USD" here? It ought to fit, and makes it super clear what currency we're using.

File: src/components/Store/Disclaimer.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
Comment (jlengstorf) [inline src/components/Store/Disclaimer.js]: As soon as the store launches, we won't need this disclaimer anymore. However, I think this is a great place to call things out (e.g. "new items!" or "thanks to @fk for making the store look great!"). With that in mind, maybe `Announcements` or `CallOut` is a better name for this component?

File: src/components/shared/Footer/About.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+`;
+
+export default () => (
+  <About>
+    <Headline>About the Gatsby Store</Headline>
+    <Content>
+      <Text>
+        The money we charge for swag helps to cover production and shipping
+        costs. In the unlikely event that Gatsby swag ends up turning a profit,
+        we'll reinvest that money into the open source community.
```
Comment (jlengstorf) [inline src/components/shared/Footer/About.js]: Typography nitpick: use `’` instead (`alt`+`shift`+`]`, unless it’s different on German keyboards).

File: src/components/shared/Footer/Footer.js [hunk 4]
```diff
@@ -1,15 +1,35 @@
...
+  margin-top: ${spacing['2xl'] * 3}px;
+  margin-bottom: ${spacing['2xl'] * 2}px;
+`;
+
+export default ({ displayAbout }) => (
+  <FooterContainer>
+    <Footer>
+      {displayAbout && <About />}
+      <LegalInfo>
+        TKTK GDPR disclosure, credits, source, any other legal info
```
Comment (jlengstorf) [inline src/components/shared/Footer/Footer.js]: For now:

> Built with 💜 by the [Gatsby Inkteam](https://www.gatsbyjs.com) · [See the source code on GitHub](https://github.com/gatsbyjs/store.gatsbyjs.org)

File: src/components/shared/Footer/metaball.svg [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1360" height="958" viewBox="0 0 1360 958">
```
Comment (jlengstorf) [inline src/components/shared/Footer/metaball.svg]: Still can't read this without screaming "MEATBALL!" in my head. 😄

File: src/fonts/Reference/How_to_use_webfonts.html [hunk 6]
```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```
Comment (jlengstorf) [inline src/fonts/Reference/How_to_use_webfonts.html]: Probably don't need the whole `Reference` folder, right?

File: src/utils/styles.js [hunk 7]
```diff
@@ -123,6 +142,55 @@ export const dropdown = {
...
+  color: ${colors.text};
+  display: block;
+  font-size: 1rem;
+  padding: ${spacing.xs}px ${spacing.sm}px;
+`;
+
+export const inputSmall = css`
+  font-size: 0.75rem;
+  padding: ${spacing.xs}px;
+`;
```
Comment (jlengstorf) [inline src/utils/styles.js]: Nitpicky, and maybe not worth it, but it seems to me like this should be wrapped up in an object like the `button` styles, e.g. `input.default`, `input.small`, `input.focus`, etc. I'll let you make the call.

File: src/utils/styles.js [hunk 8]
```diff
@@ -123,6 +142,55 @@ export const dropdown = {
...
+  )}' d='M2 0L0 2h4zm0 5L0 3h4z'/%3E%3C/svg%3E");
+  background-repeat: no-repeat;
+  background-position: right 0.5rem center;
+  background-size: 8px 10px;
+  padding-right: ${spacing.xl}px !important;
+`;
+
+export const selectSmall = css`
+  padding-right: ${spacing.xl}px !important;
+`;
```
Comment (jlengstorf) [inline src/utils/styles.js]: Same here: `select.default`, `select.small`.

This helps us establish a common pattern and also creates extremely tight relationships between related groups of styles.

File: src/utils/styles.js [hunk 9]
```diff
@@ -186,3 +254,32 @@ export const button = {
...
+
+export const buttonAsLink = css`
+  border: 0;
+  ${link};
+  background: transparent;
+  cursor: pointer;
+  padding: 0;
+  font-size: 100%;
+`;
```
Comment (jlengstorf) [inline src/utils/styles.js]: Shouldn’t this live in the `button` style object? `button.link`?

**Comment by jlengstorf** [comment]
This is looking great, @fk! I meant to put this in the review, but can we drop all the font files that aren't actually in use (e.g. all the specimen CSS, demo HTML, etc.)?

This is nitpicky, but I'd also like to see `src/fonts/futura-pt/` as the pathname, leaving out all the extra junk that was created there. I don't care _that_ much, but the organization will make it easier to find things months down the road when we need to make updates.

File: src/components/Cart/AddedToCart.js [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+export default () => (
+  <>
+    <AddedToCartMessage>
+      <strong>Excellent Choice!</strong>{' '}
+      <span role="img" aria-label="sparkles">
+        ✨
+      </span>
+      <AddedToCartProductInfo>
+        Added 1x “Purple Logo Tee”:
```
[the remark this answers is not in the document]
Comment (fk) [reply src/components/Cart/AddedToCart.js]: This currently is never shown and was meant as a placeholder for you to add your magic to—I removed the task item from #73 and added a separate issue #87. Have only thought about this for like 5 minutes and then gave up. 😺 🤷‍♂️

File: src/components/Cart/EmptyCart.js [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+export default () => (
+  <SadCartContainer>
+    <SadCart />
+    <SadCartCopy>
+      Your Cart is sad{' '}
+      <span role="img" aria-label="sad face">
+        😔
+      </span>
+      <br />
+      Go cop some purple swag!
```
[the remark this answers is not in the document]
Comment (fk) [reply src/components/Cart/EmptyCart.js]: Nice!

File: src/components/Cart/AddedToCart.js (hunk 10 above)
[the remark this answers is not in the document]
Comment (jlengstorf) [reply src/components/Cart/AddedToCart.js]: Sounds good, @fk! I'll take care of it.

[the remark this answers is not in the document]
Comment (fk) [reply src/components/Cart/AddedToCart.js]: 💪 Thank you!

File: src/utils/styles.js (hunk 7 above)
Comment (fk) [reply src/utils/styles.js]: Nah I completely agree, just didn't have the time.

Comment (fk) [reply src/utils/styles.js]: (as weird as that may sound ;-))

**Comment by fk** [comment]
Done! Just can't do much about your head screaming "MEATBALLS!" 😉

**Pull request merged**